### DISCUSSION TITLE
Add multiple charger devices

### DISCUSSION THREAD

**Discussion by JR-Y** [lbbrhzn/ocpp#1132] [General]
Hello is it somehow possible to add multiple charger devices with this integration? I would need to add 3 chargers.

Thank you!

**Comment by lbbrhzn** [comment]
Yes, that is possible. Just add the integration three times, and make sure you use different ports for each ocpp server.

BTW, we're working on a solution where you only need one instance of the integration.

**Reply by JR-Y** [reply]
Thanks for a prompt reply and the work on this integration!

**Reply by sandersr** [reply]
Hi,

I have tried that and it's not working as expected :(

My second charger appears in my integration for the 1st charger and renames my first charger to it's own name.

They are both on separate ports.

In my case both chargers are from the same manufacturer, running the same firmware, so the only difference between them is the serial number (and of course integration port).


Looks like there is a pull request which adds it:
https://github.com/lbbrhzn/ocpp/pull/1054/files

**Reply by npinguin** [reply]
can you please add support for multiple chargers from the same manuracturer

**Comment by drc38** [comment]
See https://github.com/lbbrhzn/ocpp/pull/1498 for testing with multiple chargers
